src/OData/ExpanderProjector.cs

```diff
@@ -255,6 +257,11 @@ private object Project(Field field, List<Property> expansion, List<Property> sel
                 ? ProjectMultiRefContents(refField.GetData(), expansion, selection, httpContext)
                 : (object)ProjectSingleRefContent(refField.GetData(), expansion, selection, httpContext);
         }
+        private object ProjectBinaryField(BinaryField field)
+        {
+            return RepositoryTools.GetStreamString(
+                field.Content.ContentHandler.GetBinary(field.Name).GetStream());
```

Comment:
I think we should use the same binary provider here as anywhere else when serving binary values.
```
DocumentBinaryProvider.Instance.GetStream
```